baekjoon/하노이의-탑.js

```diff
@@ -0,0 +1,14 @@
+function solution(n) {
+    const answer = [];
+    
+    function hanoi(n, from, to, other) {
+        if (n === 1) answer.push([from, to]);
+        else {
+            hanoi(n - 1, from, other, to);
+            answer.push([from, to]);
+            hanoi(n - 1, other, to, from);
+        }
```

Comment:
```suggestion
        if (!n) return;
        else {
            hanoi(n - 1, from, other, to);
            answer.push([from, to]);
            hanoi(n - 1, other, to, from);
        }
```


if-else 문에서 answer.push([from, to]) 부분이 일치하기 때문에 굳이 n===1인 경우를 따로 빼주지 않아도 똑같이 실행 가능하고, n이 0이 됐을 때만 함수 빠져나가도록 return해 주면 좋을 것 같아요.